### ISSUE TITLE
Install issues

### ISSUE THREAD

**Issue by dominicroye** [ropensci/rnaturalearth#28]
Recently I had several times problems when a student had to use the function `ne_countries (scale = 10)`.  The rnaturalearth package was installed correctly, but it required to install again without resolving the issue. It seems that the pacckage "rnaturalearthhires" fail to download or install. I don't know why. Any thoughts about this? 

Thanks!

**Comment by AndySouth** [comment]
Hola, thanks for the interest. 
I just tried this myself and it was OK. It does take a little while to install the hires package because it is big so that could possibly cause problems on a poor connection or slow PC. I recently had a similar issue on a train journey with a new PC, I had installed rnaturalearth but didn't have rnaturalearthhires and with no connection I couldn't use scale=10.
Let me know if the problem persists.

**Comment by dominicroye** [comment]
I still have problems in some cases. I just taught an introductory course and some students had problems again. Even with the rnaturalearthdata package. I had to install rnaturalearthdata directly for some students. The thing is that rnaturalearth is detecting that there is packages missing and ask to install but in the end it does not work. In all cases it was under Windows 10. Only in one case, I found that the origin was an accent in the user's name in the windows system. But it seems that this isn't the whole cause. My problem is that I can not give much more information. 

Anyway, thank you!
